pack/map.h

```diff
@@ -94,7 +94,7 @@ class Map : public IObjectMap
     const MapType& value() const;
     void           setValue(const MapType& val);
     bool           contains(const std::string& key) const;
-    const T&       operator[](const std::string& key) const;
```

Comment:
Sorry, maybe I said it wrong, but should exists 2 methods, const and nonconst...